.golangci.yml

```diff
@@ -0,0 +1,55 @@
+# Options for analysis running.
+run:
+  go: "1.19"
+  skip-files:
+    - "zz_generated.*\\.go$"
+    - "vendored_openapi\\.go$"
+  allow-parallel-runners: true
+linters:
+  disable-all: true
```

Comment:
I think we should flip it the other way. So `enable-all` and then explicitly disable linters we don't want. That way it forces us to consider new linters and whether we can them.

Reply:
on the other hand, we will be using the same linter rules as CAPI https://github.com/kubernetes-sigs/cluster-api/blob/main/.golangci.yml#L13 this way

Reply:
Make sense

Reply:
Yes @alexander-demicev initially I took the idea from CAPI templates

Reply:
I don't think we need to follow CAPI here, as a provider, we can choose what linters to enable. It's far better to get the benefit of new linters and then have to make an explicit decision to disable one. If you use the disable all then you are not forced to evaluate new linters.

Reply:
Resolving this as I have changed to enable all.